engine/player/unique_gear_dragonflight.cpp

```diff
@@ -9829,6 +9990,7 @@ void register_special_effects()
   register_special_effect( 422479, items::infernal_signet_brand );
   register_special_effect( 429228, items::rezans_gleaming_eye );
   register_special_effect( 421990, items::gift_of_ursine_vengeance );
+  register_special_effect( 422750, items::fyrakks_tainted_rageheart );
```

Comment:
Should also disable the equip driver below around line 10083 to be sure simc never auto parses it and messes with the implementation if theres any spell data changes.

Reply:
I'm not sure what you mean by equip driver. Is this line incorrect?

Reply:
so the trinket has 2 drivers, one is the equip effect, that does AoE damage constantly, id `422652`. 

usually when we custom write these types of effects, we disable that driver so that nothing funny happens with spell data changes. look at like 10083 for some examples.